Gems/PhysX/Code/Tests/StaticRigidBodyComponentTests.cpp

```diff
@@ -51,36 +51,57 @@ namespace PhysXEditorTests
             boxDimensions);
     }
 
-    TEST_F(PhysXEditorFixture, StaticRigidBodyComponent_NoRigidBody_RuntimeStaticRigidBodyComponentCreated)
+    TEST_F(PhysXEditorFixture, StaticRigidBodyComponent_NoRigidBody_NoRuntimeStaticRigidBodyComponent)
+    {
+        // Create editor entity
+        EntityPtr editorEntity = CreateInactiveEditorEntity("Entity");
+        editorEntity->CreateComponent<PhysX::EditorShapeColliderComponent>();
+        AddEditorBoxShapeComponent(editorEntity);
+
+        // Create game entity and verify StaticRigidBodyComponent was NOTcreated
+        EntityPtr gameEntity = CreateActiveGameEntityFromEditorEntity(editorEntity.get());
+        const auto* staticRigidBody = gameEntity->FindComponent<PhysX::StaticRigidBodyComponent>();
+
+        EXPECT_TRUE(staticRigidBody == nullptr);
```

Comment:
nit: It may be useful for EXPECT_ for some situations to surface the cause of failure 
```suggestion
        EXPECT_TRUE(staticRigidBody == nullptr) << "A StaticRigidBodyComponent should NOT have been created";
```

Reply:
It's a good suggestion, I'll keep it in mind for future usages. It could be done later as a clean up pass (since all tests in the document do not include that extra message), but this work change is already 5 PRs, so I'd like to avoid making large clean up changes at the same time to keep it manageable. Thanks.